lib/default_settings.dart

```diff
@@ -0,0 +1,22 @@
+/// The default output directory
+const defaultOutputDirectory = 'lib';
```

Comment:
Because these constants are not part of a specific Constant class, I find it a little bit strange to call them directly in the other
May we put them in uppercase ?
That way, in other files that need these constants, we would know, juste by seeing it, that they are constants from somewhere

Reply:
Yes here they could be in a class DefaultSettings which would make the code more readable, I was following the example of `template.dart` where the template const string was declared outside of a class. Will update now.

With regard to lowerCamelCase or UPPER_CASE, I am happy to change as mentioned in the point above.

Reply:
Hi, I think that it is you that had the better idea following the documentation

I forgot that, in dart, const are in lowerCase.
So, in order to be readable AND to do it the Dart's way, maybe the 2 option would be the best :)

Reply:
Moved all consts inside DefaultSettings class.

Reply:
Moved all consts into DefaultSettings class.